lib/shoulda/matchers/active_model/strict_validator.rb

```diff
@@ -0,0 +1,50 @@
+module Shoulda
+  module Matchers
+    module ActiveModel
+      module StrictValidator
+        def allow_description(allowed_values)
+          "doesn't raise when #{attribute} is set to #{allowed_values}"
+        end
+
+        def expected_message_from(attribute_message)
+          "#{human_attribute_name} #{attribute_message}"
+        end
+
+        def formatted_messages
+          [messages.first.message]
+        end
+
+        def messages_description
+          if has_messages?
+            ': ' + messages.first.message.inspect
+          else
+            ' no exception'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.